controllers/baremetalset_controller.go

```diff
@@ -440,13 +473,16 @@ func (r *BaremetalSetReconciler) baremetalHostProvision(instance *ospdirectorv1b
 	}
 
 	sts = append(sts, userDataSt)
+	ip, network, _ := net.ParseCIDR(ipset.Status.HostIPs[bmhName].IPAddresses["ctlplane"]) // We use ctlplane as the MgmtNetwork too for now
+
+	r.Log.Info(fmt.Sprintf("Network: %s", network.String()))
```

Comment:
Is this left here intentionally?  If so, might be good to include BaremetalSet name and `bmhName` to identify the association with the network.